compiler/codegen/OMRAheadOfTimeCompile.cpp

```diff
@@ -115,6 +116,26 @@ void OMR::AheadOfTimeCompile::traceRelocationOffsets(uint8_t *&cursor, int32_t o
       }
    }
 
+
+uint8_t* OMR::AheadOfTimeCompile::initializeAOTRelocationHeader(TR::IteratedExternalRelocation *relocation)
+   {
+   TR_UNIMPLEMENTED();
```

Comment:
could you please add some documentation to these functions so we know what belongs here when extended?

Reply:
I have copied the comments from similar OpenJ9 methods and left the link in the header file. Will that be enough?

Reply:
Thank you for your comment @ymanton , I have added some comments to ``processRelocations``. ``dumpRelocationData``, ``traceRelocationOffsets`` and ``interceptAOTRelocation`` were added by another developer and I am not entirely certain what it needs to look like. I will push the updated branch with the comments shortly